iphone/Classes/TiUITabGroup.m

```diff
@@ -402,6 +402,12 @@ -(void)setUnselectedItemTintColor_:(id)value
 }
 #endif
 
+-(void)setTabsTranslucent_:(id)value
+{
+    BOOL tabsTranslucent =  [TiUtils boolValue:value def:YES];
+    controller.tabBar.translucent = tabsTranslucent;
+}
```

Comment:
Rename to `translucent`, no idea why we ever started prefixing it with `tabs` some properties, it should be more unified. So in your case, it would just be `setTranslucent` instead.